### PR TITLE
Translate 1.9.3 EOL article (ja)

### PR DESCRIPTION
**Pull request by makimoto**
@ruby/www-ruby-lang-org-i18n-ja :eyeglasses: ?

### CODE REVIEW

File: ja/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+author: "Olivier Lacan"
+translator: "makimoto"
+date: 2015-02-23 00:00:00 +0000
+lang: ja
+---
+
+本日 (2015年2月23日) をもって Ruby 1.9.3 のすべてのサポートが終了しました。
+今後、新しいバージョンの Ruby で行われたバグ修正やセキュリティ修正は、1.9.3 にバックポートされません。
+
+このサポート終了は[1年以上前に告知されたもの](https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/)です。
```
Comment (JuanitoFatas) [inline ja/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/

=>

https://www.ruby-lang.org/ja/news/2014/01/10/ruby-1-9-3-will-end-on-2015/

:smile:

File: ja/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+できるだけすぐに Ruby 2.0.0 以上にアップグレードすることを、われわれは強く推奨します。
+1.9.3 ブランチのメンテナンスを継続したい方や、なんらかの理由でアップグレードができない方は、ご連絡ください
```
Comment (sorah) [inline ja/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: no 。 at the end

File: ja/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md (hunk 1 above)
Comment (makimoto) [inline ja/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: oops... (force updated)

File: ja/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+lang: ja
+---
+
+本日 (2015年2月23日) をもって Ruby 1.9.3 のすべてのサポートが終了しました。
+今後、新しいバージョンの Ruby で行われたバグ修正やセキュリティ修正は、1.9.3 にバックポートされません。
+
+このサポート終了は[1年以上前に告知されたもの](https://www.ruby-lang.org/ja/news/2014/01/10/ruby-1-9-3-will-end-on-2015/)です。
+
+できるだけすぐに Ruby 2.0.0 以上にアップグレードすることを、われわれは強く推奨します。
+1.9.3 ブランチのメンテナンスを継続したい方や、なんらかの理由でアップグレードができない方は、ご連絡ください
```
Comment (sorah) [inline ja/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: no 。 at the end

Comment (makimoto) [inline ja/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: fixed. thanks.

**Comment by sorah** [comment]
dupe comment...

**Pull request merged**